tools/ExtensionLibrary/DevHome.ExtensionLibrary/ViewModels/ExtensionLibraryViewModel.cs

```diff
@@ -47,15 +53,59 @@ public async Task GetUpdatesButtonAsync()
         await Launcher.LaunchUriAsync(new ("ms-windows-store://downloadsandupdates"));
     }
 
+    [RelayCommand]
+    public async Task LoadedAsync()
+    {
+        await GetInstalledExtensionsAsync();
+        GetAvailablePackages();
+    }
+
     private async void OnPluginsChanged(object? sender, EventArgs e)
     {
-        await _dispatcher.EnqueueAsync(() =>
+        await _dispatcher.EnqueueAsync(async () =>
         {
             ShouldShowStoreError = false;
+            await GetInstalledExtensionsAsync();
             GetAvailablePackages();
         });
     }
 
+    private async Task GetInstalledExtensionsAsync()
+    {
+        var extensionWrappers = await _pluginService.GetInstalledPluginsAsync(true);
+
+        InstalledPackagesList.Clear();
+
+        foreach (var extensionWrapper in extensionWrappers)
+        {
+            // Don't show self as an extension.
+            if (Package.Current.Id.FullName == extensionWrapper.PackageFullName)
+            {
+                continue;
+            }

```

Comment:
Can use the other overload of GetInstalledExtensionsAsync() with provider trype. https://github.com/microsoft/devhome/blob/b37b1d4697d55f60e776538ac452abae496b807e/src/Services/PluginService.cs#L223

Reply:
Oh I see, you don't want to ignore the ones without ISettingsProvider. Ignore comment.